lib/routes/gov/stats/index.ts

```diff
@@ -108,18 +108,14 @@ async function handler(ctx) {
                     return item;
                 }
 
-                try {
-                    item.author = detailResponse.data.match(/来源：(.*?)</)[1].trim();
-                } catch {
-                    item.author = content('div.detail-title-des h2 span').first().text().split(':').pop().trim();
-                }
+                item.author = detailResponse.data?.match(/来源：(.*?)</)?.[1]?.trim() ?? detailResponse.match(/来源：(.*?)</)?.[1]?.trim();
```

Comment:
The first option is always false, as it is the key differences between `got` and `ofetch` utility. `got` returns the HTML in `.data` while `ofetch` returns the HTML directly. This is one of the left behind during refactor in https://github.com/DIYgod/RSSHub/commit/4f678bd7974919fe99c0402ef6331af1986baa5b